packages/dashmate/README.md

```diff
@@ -259,7 +260,7 @@ The [setup](#setup-node) command set corresponding group as default. To output t
 
 ```
 USAGE
-  $ dashmate group default [GROUP]
+  $ dashmate group:default [GROUP]
```

Comment:
On my machine, dashmate 0.22.13 shows the group commands as they were before this PR (i.e. without requiring the `:`). See the help output:

```
dashmate group
Orchestrate group of nodes

USAGE
  $ dashmate group COMMAND

COMMANDS
  group default  Manage default group
  group list     List available groups
  group reset    Reset group nodes
  group restart  Restart group nodes
  group start    Start group nodes
  group status   Show group status overview
  group stop     Stop group nodes
```

Reply:
Thanks for pointing it out. After talk with Ivan I made changes to all legacy command forms (from ":" to " ").